live-examples/js-examples/object/object-entries.html

```diff
@@ -1,14 +1,20 @@
 <pre>
-<code id="static-js">const object1 = { foo: 'bar', baz: 42 };
-console.log(Object.entries(object1)[1]);
-// expected output: Array ["baz", 42]
+<code id="static-js">const object = { 'foo': 'a', 'bar': 2};
```

Comment:
We use multiple lines for this: https://github.com/mdn/interactive-examples/blob/master/JS-Example-Style-Guide.md#property-definition (I appreciate the code was originally like this, but it's worth fixing I think).